students/hirot_directory/session05/mailroom5.py

```diff
@@ -116,30 +96,28 @@ def send_letters():
             f.close()
 
     except IOError as i:
-        print("Error code is", i)
-    print("letter drafts are in the folder for your review. Please review before mailing out.")
+        print("Sorry...Program crashed....Error type is...", type(i))
 
+    else:
+      print("letter drafts are in the folder for your review. Please review before mailing out.")
+
+
+dict_answer = {'1': thank_you_loop, '2': print_list, '3': send_letters, '4': quit}
 
-dict_answer = {1: thank_you_loop, 2: create_report, 3: send_letters, 4: quit}
 
 def mainloop():
     while True:
         try:
-            answer = int(input("Select from one of these options:\n"
+            answer = input("Select from one of these options:\n"
                                "(1) Send a Thank you\n"
                                "(2) Create a Report\n"
                                "(3) Send letters to everyone\n"
                                "(4) quit\n"
-                               ))
+                               )
             dict_answer[answer]()
 
-        except ValueError as e:
-            print("Error happened. Error is ", type(e), "Please type 1, 2 or 3")
         except KeyError as k:
-            print("Error happened. Error is ", type(k), "Please type 1, 2 or 3")
-        # finally:
-        #   print("please type 1, 2, or 3")
-
+            print("Sorry...Program crashed....Error type is....", type(k), "Please type 1, 2 or 3")
```

Comment:
did the program crash? or did you catch the error? it can only be one or the other....